test/api/test_relation_api.cpp

```diff
@@ -369,6 +385,15 @@ TEST_CASE("Test crossproduct relation", "[relation_api]") {
 	REQUIRE(CHECK_COLUMN(result, 1, {10, 10, 10, 5, 5, 5, 4, 4, 4}));
 	REQUIRE(CHECK_COLUMN(result, 2, {1, 2, 3, 1, 2, 3, 1, 2, 3}));
 	REQUIRE(CHECK_COLUMN(result, 3, {10, 5, 4, 10, 5, 4, 10, 5, 4}));
+
+	// run a positional cross product
+	auto join_ref_type = JoinRefType::POSITIONAL;
+	vcross = v1->CrossProduct(v2, join_ref_type);
```

Comment:
I'm not sure what you mean, there are join_types and join_ref types. Do you mean just make it  a regular join? It would look something like 
`vcross = v1->Join(v2, "v1.i=v2.j", JoinType::INNER, join_ref_type);`

Reply:
Something like `v1->PositionalJoin(v2)` would make more sense to me

Reply:
hmm, I feel like adding the type of join to function name might add unnecessary overhead/maintenance. If we do, we may also end up creating relational functions like `v1->InnerJoin(v2)`, `v1->LeftJoin(v2)` , `v1->AsOfJoin(v2)` etc. Since the `CrossProduct` relation already exists I think we should keep someone is already using it. But I think we should stick with 
`vcross = v1->Join(v2, "v1.i=v2.j", JoinType::INNER, JoinRefType::POSITIONAL)` for positional joins